pages/common/nkf.md

```diff
@@ -0,0 +1,27 @@
+# nkf
+
+> Convert character code and new line code.
```

Comment:
Please add "Network kanji filter" as the main heading.

From the man page here https://linux.die.net/man/1/nkf, it seems that it converts input kanji code from one encoding to another. I think that is more clear. The current description does not mention what does it convert to.